pallets/gear/src/benchmarking/code.rs

```diff
@@ -85,11 +85,6 @@ pub struct ModuleDefinition {
     /// The amount of I64 arguments the aux function should have.
     pub aux_arg_num: u32,
     pub aux_res: Option<ValueType>,
-    /// If set to true the stack height limiter is injected into the the module. This is
-    /// needed for instruction debugging because the cost of executing the stack height
-    /// instrumentation should be included in the costs for the individual instructions
-    /// that cause more metering code (only call).
-    pub inject_stack_metering: bool,
```

Comment:
Why it has been removed?

Reply:
No need

Reply:
`inject_stack_metering()` is unused and should be removed then